storage/ndb/rest-server2/server/src/feature_store_data_structs.hpp

```diff
@@ -93,44 +93,44 @@ class BatchFeatureStoreRequest {
   std::string to_string() const {
     std::ostringstream oss;
     oss << "BatchFeatureStoreRequest {"
-        << "\n  featureStoreName: " << featureStoreName
-        << "\n  featureViewName: " << featureViewName
-        << "\n  featureViewVersion: " << featureViewVersion
-        << "\n  passedFeatures: {";
+        << "\nfeatureStoreName: " << featureStoreName
```

Comment:
one can remove all white characters to save some bandwidth

Reply:
Yep, removed a significant amount of whitespace.